cinema/migrations/0002_alter_movie_description.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.1.4 on 2024-12-06 13:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='movie',
+            name='description',
+            field=models.CharField(max_length=255),
```

Comment:
This migration changes the `description` field to a `CharField` with `max_length=255`. If the `description` field in the `Movie` model is intended to be a `TextField`, you should revert this change to maintain consistency between the model and its serializer.